resources/wiki-templates/overview.md

```
@@ -0,0 +1,33 @@
+---
+tags:
+  - topic/meta
+---
+
+# 📦 Template: repository (agent-memory)
+
+Repository notes live in `Projects/<Project>/<repo>/`. Reference: `Projects/MySky/custom-pricelist-api/`.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)